flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -269,9 +269,10 @@ public class KubernetesConfigOptions {
     @Documentation.OverrideDefault(
             "The default value depends on the actually running version. In general it looks like \"flink:<FLINK_VERSION>-scala_<SCALA_VERSION>\"")
     public static final ConfigOption<String> CONTAINER_IMAGE =
-            key("kubernetes.container.image")
+            key("kubernetes.container.image.ref")
```

Comment:
There does not appear to be a proper term for this, so I used "ref" similar to what the log4j uses.